src/libraries/System.Linq.Parallel/README.md

```diff
@@ -0,0 +1,16 @@
+# System.Linq.Parallel
+
+Documentation can be found at https://learn.microsoft.com/dotnet/api/system.linq.
+
+Parallel LINQ (PLINQ) is a parallel implementation of the Language-Integrated Query (LINQ) pattern. PLINQ implements the full set of LINQ standard query operators as extension methods for the System.Linq namespace and has additional operators for parallel operations.
+
+## Contribution Bar
+
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
```

Comment:
I'm not sure if PLINQ accepts new features anymore. cc @stephentoub

Reply:
Fwiw Git history doesn't show new features in over 2 years so I'd say maintenance mode.

Reply:
Seems you're right changing to: `- [x] [We only consider lower-risk or high-impact fixes to maintain or improve quality](../../libraries/README.md#primary-bar)`